modules/ROOT/pages/_partials/include-nav-steps-telemetry-permissions.adoc

```diff
@@ -0,0 +1,17 @@
+// tag::connectionPermission[]
+. Log in to Anypoint Platform using an account that has the Telemetry Exporter Administrator permission.
+. In the sidebar or the main Anypoint Platform screen, click *Monitoring*.
+. Click *Telemetry Exporter*.
+// end::connectionPermission[]
+
+// tag::configurationPermission[]
+. Log in to Anypoint Platform using an account that has the Telemetry Exporter Configuration Manager permission.
```

Comment:
The permission needs to be renamed `Telemetry Exporter Configuration Manager` -> `Telemetry Exporter Configurations Manager` throughout.